mkdocs/docs/HPC/web_portal_custom_apps/abaqus_cae.md

```diff
@@ -1,3 +1,28 @@
 # Custom web portal app for ABAQUS (CAE course)
 
-*(more info soon)*
+Cluster Desktop with Abaqus - course E071400
+This app will launch an interactive desktop on a donphan node with 2 cores.
+It will then start the Abaqus GUI after setting some environment variables.
+
+Locked down options (in form.yml.erb):
+-	Xfce desktop
+-	1 node, 2 cores
+-	Cluster donphan
+-	Free selection 1-8 hours
```

Comment:
Is this useful info? Especially the `form.yml.erb` is meaningless to end users, and rather than mentioning `donphan` directly, we should mention the more general "interactive/debug cluster" instead?

Also, this doesn't really add anything compared to what's above, so just remove?

```suggestion
```